src/node_file.cc

```diff
@@ -721,7 +721,7 @@ void AfterScanDir(uv_fs_t* req) {
   int r;
   std::vector<Local<Value>> name_v;
 
-  for (int i = 0; ; i++) {
+  while (true) {
```

Comment:
Can we convert this to `for(;;)` so that it remains consistent with the coding style.
```suggestion
  for (;;) {
```

Also at other places as well.

Reply:
I search in the project with `for (;;)` or `while (true)`, their counts are almost the same.
I adopt the opinion but not really into it

Reply:
The reason for this being, this coding style is followed in libuv as well as in other c++ files that are interacting in the C++ layer or to be consistent with libuv style. However open to discussion.

Reply:
makes sense, good suggestions.